duckduckgo/iOS
# Check if problem with Auto Clear overlay occurs and try to fix it if needed

Task/Issue URL: https://app.asana.com/0/414709148257752/1199525606019720

**Description**:
In case Blank Overlay has not been dismissed, report that as an error and try to fix this problem.

**Steps to test this PR**:
1. Tweak the code to ensure overlay is not removed (basically return early from `private func removeOverlay()`).
2. Run the app, enable Auto Clear data.
3. Background the app.
4. Put it up again - blank overlay should be displayed.
5. Press any button - that state should be fixed and pixel should be fired.



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
